WalletWasabi/Legal/Assets/LegalDocuments.txt

```diff
@@ -257,12 +263,9 @@ COMPLIANCE WITH TAX OBLIGATIONS
 COUNTRY OF RESIDENCE
 ====================
 
-    In case You are a Gibraltar resident and willing to use Wasabi Wallet please inform us in advance on the following e-mail address: legal@zksnacks.com.
+In case You are a Gibraltar or Republic of Seychelles resident and willing to use Wasabi Wallet please inform us in advance on the following e-mail address: legal@zksnacks.com.
```

Comment:
```suggestion
    In case You are a Gibraltar or Republic of Seychelles resident and willing to use Wasabi Wallet please inform us in advance on the following e-mail address: legal@zksnacks.com.
```

Why here we mention both Gibraltar and Republic of Seychelles?